pywps/exceptions.py

```diff
@@ -34,7 +34,7 @@ class NoApplicableCode(HTTPException):
     Base exception class
     """
 
-    code = 400
+    code = 500
```

Comment:
400 -> NotFound
500 -> ServerError 

which on this place is IMHO more appropriate (no applicable exception)